src/server/src/main/java/io/cassandrareaper/ReaperApplicationConfiguration.java

```diff
@@ -65,6 +65,11 @@
   @DefaultValue("false")
   private Boolean incrementalRepair;
 
+  @JsonProperty
+  @NotNull
+  @DefaultValue("true")
+  private Boolean blacklistTwcsTables;
```

Comment:
We're moving towards (thinking about?) keeping application-state in the storage, and keeping the configuration purely about Reaper's configuration (and not about the configuration of clusters that a user registers in a running Reaper).
That change hasn't materialised yet, but am noting it here for forward-reference sake.

Reply:
That’s great to hear.  That will definitely make the app config cleaner. Plus, supporring multiple clusters is tricky for us since we need to update reaper config and cycle when new clusters are added (all contain their own unique jmx creds).